inject/src/main/java/io/micronaut/context/env/AbstractPropertySourceLoader.java

```diff
@@ -152,4 +157,22 @@ protected void processMap(Map<String, Object> finalMap, Map map, String prefix)
             }
         }
     }
+
+    /**
+     * Return logEnabled value.
+     *
+     * @return is log enabled
+     */
+    public boolean isLogEnabled() {
+        return logEnabled;
+    }
+
+    /**
+     * Setter for logEnabled.
+     *
+     * @param logEnabled is log enabled
```

Comment:
```suggestion
     * @param logEnabled is log enabled
     * @since 3.9.0    
```

Same in `PropertySourcePropertyResolver`

Reply:
fixed